toolkits/springernature/packages/springernature-forms/scss/50-components/_errors.scss

```diff
@@ -0,0 +1,31 @@
+
+.c-forms__error-summary {
+	border: $forms--errors-summary-border;
+	padding: $forms--errors-summary-padding;
+	margin-bottom: $forms--errors-summary-margin-bottom;
+}
+
+.c-forms__error-summary-heading {
+	margin-bottom: $forms--errors-summary-heading-margin-bottom;
+}
+
+.c-forms__error-summary-general-list,
```

Comment:
what is the difference between `c-forms__error-summary-general-list` and `c-forms__error-summary-specific-list`? I don't see an example of the former in the readme or the demo. This should be detailed at least in the readme

Reply:
True, thank you. General mean it is not related with a specific field from the form, maybe due to a combination of fields, or a constraint... a specific error is bound to one field.

I will add an example to the Readme 💯